tinkerbell/rufio/templates/secrets-viewer-role-binding.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.rufio.deploy }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
Because we're using a roll binding permissions for secrets will be restricted to the namespace the role binding is deployed to. 

There was a patch recently in Rufio to correct this (I think it was just making it a cluster role binding).

Reply:
do you have a link, by chance? im only seeing this https://github.com/tinkerbell/rufio/blob/main/config/rbac/secrets_viewer_role_binding.yaml and its using `RoleBinding`.

Reply:
Hey @chrisdoherty4 , I've updated the rufio charts. when you have a moment, please check it out. Thanks!